src/Playwright/Transport/Connection.cs

```diff
@@ -195,6 +195,7 @@ internal static async Task InstallAsync(string driverPath = null, string browser
 
                 string messageString = JsonSerializer.Serialize(message, GetDefaultJsonSerializerOptions());
                 _logger?.LogInformation($"pw:channel:command {messageString}");
+                Console.WriteLine(messageString);
```

Comment:
```suggestion
                System.Diagnostics.Debug.WriteLine(messageString);
```

If we want to keep it, we can use the `Debug` output,  which gets ignored for `Release` builds, but it can offer insight for us. Otherwise, we should remove it.

Reply:
Let's drop it.